libsolidity/formal/SSAVariable.cpp

```diff
@@ -39,7 +40,10 @@ SSAVariable::SSAVariable(
 		m_symbolicVar = make_shared<SymbolicBoolVariable>(_decl, _interface);
 	else
 	{
-		solAssert(false, "");
+		// Use a fallback type. This allow the SMT checker to run on functions
+		// which use unsupported types. It may introduce false-positivies, but
+		// it will allows the checker to run on the known types.
+		m_symbolicVar = make_shared<SymbolicFallbackVariable>(_decl, _interface);
```

Comment:
Would it be worth issuing a warning here with a condensed version of the above text?

Reply:
The warning is emitted in the [`createVariable` function](https://github.com/ethereum/solidity/pull/4949/files#diff-8ee88fd6f33965616a4a99fe36c2ba48R792)